packages/fleather/lib/src/widgets/text_selection.dart

```diff
@@ -1,6 +1,4 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
Dragging collapsed selection handle opens magnifier both in native widgets and Flutter's `TextField` but not in current implementation of this PR.

Reply:
nice catch, let me check